docs/gui/modlistman.rst

```diff
@@ -0,0 +1,17 @@
+gui/modlistman
+===========
+
+.. dfhack-tool::
+    :summary: Simple modlist manager.
+    :tags: dfhack interface
+
+Adds an optional overlay to the mod list screen that
+allows you to save and load mod list presets, as well
+as set a default mod list preset for new worlds.
+
+Usage
+-----
+
+::
+
+    gui/modlistman
```

Comment:
right now, there's nothing that happens when the script is "run" (since the entrypoint is currently just the overlay). it would be useful if this script provided a list of what mods are currently enabled when run when a fort is loaded, but that doesn't need to be done as part of this PR.

Reply:
i've looked into grabbing the modlist of an active fort but came up with squat, unfortunately

Reply:
I've found a reliable list in `df.global.world.object_loader`, which is what I use to dynamically generate the script path for mods.

Some code here might be refactorable: https://github.com/DFHack/dfhack/blob/develop/library/lua/script-manager.lua#L135